packages/server/src/index.ts

```diff
@@ -35,6 +35,7 @@ app.get('/cron/daily_total_sign_ups', async (req, res) => {
   if (req.get('X-Appengine-Cron')) {
     try {
       await updateTimeSeries()
+      await createOrUpdateAnalyticsDashboard()
```

Comment:
We run it in the `cronjob` because it allows us to automatically update the dashboard as new orgs/states are added to the app.